pkg/tasks/component.go

```diff
@@ -0,0 +1,106 @@
+package tasks
+
+import (
+	"context"
+	"database/sql"
+
+	cs "github.com/cloudtrust/common-service"
+	"github.com/cloudtrust/common-service/database"
+	"github.com/cloudtrust/common-service/log"
+	api "github.com/cloudtrust/keycloak-bridge/api/tasks"
+	"github.com/cloudtrust/keycloak-bridge/internal/keycloakb"
+	"github.com/cloudtrust/keycloak-client"
+)
+
+// Component interface exposes methods used by the bridge API
+type Component interface {
+	GetActions(ctx context.Context) ([]api.ActionRepresentation, error)
+	DeleteDeniedTermsOfUseUsers(ctx context.Context) error
```

Comment:
It is still the old naming

Reply:
We did not speak about a new name here... GetExpiredTermsOfUseAcceptance would no make sense here.
CleanUpAccordingToExpiredTermsOfUseAcceptance ?

Reply:
DeleteUsersWithExpiredTermsOfUseAcceptance ?